api/src/pcapi/tasks/external_api_booking_notification_tasks.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from pcapi import settings
+from pcapi.routes.serialization import BaseModel
+from pcapi.tasks.decorator import task
+from pcapi.tasks.serialization.external_api_booking_notification_tasks import ExternalApiBookingNotificationRequest
+from pcapi.utils import requests
+
+
+logger = logging.getLogger(__name__)
+
+
+class ExternalApiBookingNotificationTaskPayload(BaseModel):
+    data: ExternalApiBookingNotificationRequest
+    notificationUrl: str
+
+
+@task(settings.GCP_EXTERNAL_API_BOOKING_NOTIFICATION_QUEUE_NAME, "/external_api/booking_notification")
+def external_api_booking_notification_task(payload: ExternalApiBookingNotificationTaskPayload) -> None:
+    try:
+        response = requests.post(payload.notificationUrl, json=payload.data)
+        response.raise_for_status()
+    except (requests.exceptions.HTTPError, requests.exceptions.RequestException) as exception:
```

Comment:
`HTTPError` dérive de `RequestException`, donc en attrapant la seconde (la classe de base), tu attrapes aussi la première : 

```suggestion
    except requests.exceptions.RequestException as exception:
```